README.md

```diff
@@ -10,21 +10,65 @@ npm install @bundled-es-modules/pdfjs-dist
 
 ## Use
 
-```html
-<script type="module">
-  import { pdfjs } from '@bundled-es-modules/pdfjs-dist/index.js';
-  pdfjs.GlobalWorkerOptions.workerSrc = '@bundled-es-modules/pdfjs-dist/pdf.worker.js';
-  var url = 'basicapi.pdf';
-  var loadingTask = pdfjs.getDocument(url);
-  //...
-</script>
+```js
+import pdfjs from "@bundled-es-modules/pdfjs-dist/build/pdf";
+import viewer from "@bundled-es-modules/pdfjs-dist/web/pdf_viewer";
+
+pdfjs.GlobalWorkerOptions.workerSrc =
+  "@bundled-es-modules/pdfjs-dist/pdf.worker.js";
+
+var url = "basicapi.pdf";
+var loadingTask = pdfjs.getDocument(url);
+//...
 ```
 
-## Overview of this mirror repo
+**Important**: Unlike in the original build, the `pdfjs.GlobalWorkerOptions.workerSrc` is mandatory.
+
+# Overview
+
+### Background
+
+PDFJS is a library by Mozilla, created to facillitate client-side rendering of PDF documents.
```

Comment:
This new background section is informative: UMD vs ESM. Do you think that the original summary above might be sufficient?  I sort of feel like "if you're here, you know why"?

Reply:
Shortened the background section to simply state what PDFJS is, and describe the original repository we're mirroring.